sos/report/plugins/gluster.py

```diff
@@ -72,7 +74,10 @@ def setup(self):
             "/etc/glusterfs",
             "/var/lib/glusterd/",
             # collect nfs-ganesha related configuration
-            "/run/gluster/shared_storage/nfs-ganesha/"
+            "/run/gluster/shared_storage/nfs-ganesha/",
+            # collect status files and public ssh keys
+            "/var/lib/glusterd/.keys/",
```

Comment:
Are we absolutely, 100% sure, that _only_ public keys are stored here and that no private key _ever_ is stored here by the service?